boefjes/boefjes/plugins/kat_snyk/main.py

```diff
@@ -12,7 +13,7 @@ def run(boefje_meta: BoefjeMeta) -> list[tuple[set, bytes | str]]:
     software_name = input_["name"]
     software_version = input_["version"]
 
-    result = {
+    result: dict[str, Any] = {
```

Comment:
Based on the code I assumed the dictionary values to be lists instead of any type of value. Perhaps this is a more correct type? 

```suggestion
    result: dict[str, list[Any]] = {
```

Reply:
The values are a list of dicts, so I've made it even more strict with `dict[str, list[dict]]`

Reply:
That's even better